src/legionpe/theta/utils/ReportErrorTask.php

```diff
@@ -38,19 +38,27 @@ public function onRun(){
 		if(microtime(true) - self::$last0 < 60){
 			return;
 		}
-		self::log("ping-all !!! PEMapModder ping!");
-		self::log("Exception caught: " . $this->exMsg);
-		self::log("In file: " . $this->exFile . "#" . $this->exLine);
-		self::log("Happened during: " . $this->when);
-		self::log("On the server below:");
-		Utils::getURL(Credentials::IRC_WEBHOOK_NOPREFIX . urlencode("BotsHateNames: status " . Settings::$LOCALIZE_IP . " " . Settings::$LOCALIZE_PORT));
+		$message = "=== ERROR BEGIN ===".PHP_EOL;
```

Comment:
If you're using plain-text hooks it'll just be converted into one message per newline, subject to the normal rate limiting.